lib/measured/measurable.rb

```diff
@@ -66,4 +68,20 @@ def name
     end
 
   end
+
+  private
+
+  def value_string
+    @value_string ||= begin
+      str = case value
+      when Rational
+        value.denominator == 1 ? value.numerator.to_s : value.to_f.to_s
+      when BigDecimal
+        value.to_s("F")
+      else
+        value.to_f.to_s
+      end
+      str.gsub(/\.0*\Z/, "")
```

Comment:
I'm pretty confident it would be the same as [`printf` specification](http://pubs.opengroup.org/onlinepubs/9699919799/functions/printf.html), where it defaults to a precision of 6 unless explicitly given.